i18n/ja/docusaurus-plugin-content-docs/current/unraid-os/release-notes/6.12.0.md

```
@@ -61,7 +61,7 @@ macvlan に関連する call trace が発生している場合、まず _**Setti
 
 しばらくすると Docker コンテナの起動に問題が発生し、Plex を使用している場合は、Plex の Docker コンテナ設定を開き、詳細表示に切り替えて、Extra Params に `--no-healthcheck` を追加してください
 
-### ロールバック {/* #rolling-back */}
+### ロールバック {/* #rolling-back */} {/* #rolling-back */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)